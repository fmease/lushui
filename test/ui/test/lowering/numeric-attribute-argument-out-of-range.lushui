;;; TEST fail check

@(recursion-limit 130'454'112'009'021)
module dummy of
