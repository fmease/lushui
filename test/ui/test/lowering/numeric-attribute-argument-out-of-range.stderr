error[E008]: attribute argument does not fit integer interval [0, 2^32-1]
  ┌─ ${TEST_FOLDER}/lowering/numeric-attribute-argument-out-of-range.lushui:1:19
  │
1 │ @(recursion-limit 130'454'112'009'021)
  │                   ═══════════════════
  │

error: aborting due to previous error

